src/index.rs

```diff
@@ -1708,6 +1738,19 @@ impl Index {
             .collect::<Vec<_>>()
     }
 
+    /// Returns all declared parameters of a POU, including those defined in super-classes
+    pub fn get_declared_parameters_2nd(&self, pou: &str) -> Vec<&VariableIndexEntry> {
```

Comment:
see if we can merge this with get_declared_parameters, otherwise rename to get_declared_parameters_recursive or something